integration/examples_nodejs_test.go

```diff
@@ -82,6 +82,15 @@ func TestLogs(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestCloudFront(t *testing.T) {
+	test := getJSBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir: filepath.Join(getCwd(t), "cloudfront"),
```

Comment:
Would it be possible to add a quick assert, even if it's hello-world style, that some output of the stack got returned correctly? Thanks!